### PR TITLE
Test demos in travis checks

### PR DESCRIPTION
**Pull request by otraore**
This will truly ensure that demos can be built when testing (this will be really useful for PRs).

https://www.reddit.com/r/golang/comments/3393hg/go_build_doesnt_build_binaries/

> When the command line specifies a single main package, build writes the resulting executable to output. Otherwise build compiles the packages but discards the results, serving only as a check that the packages can be built.
>

### CODE REVIEW

**Comment by EtienneBruines** [comment]
You do know they are already being built by travis (by default) ? 

But I guess it helps with #249

**Comment by mewmew** [comment]
Potentially, we could also make sure that the tutorial ([EngoEngine/TrafficManager](https://github.com/EngoEngine/TrafficManager)) is building, even if it is located in a separate repository.

File: .travis.yml [hunk 1]
```diff
@@ -8,3 +8,6 @@ before_install:
 go_import_path: engo.io/engo
 go:
 - 1.6.2
+script:
+    - go test -v ./..
```
Comment (mewmew) [inline .travis.yml]: Should be `go test -v ./...` (missing one dot `.`).

**Comment by paked** [comment]
@mewmew That is a good idea. Lets get TrafficManager in on the travis-ci fun!

**Comment by otraore** [comment]
"You do know they are already being built by travis (by default) ?" - @EtienneBruines 

To the best of my understanding, no they are not. Only tests are being ran as far as I know.

**Comment by EtienneBruines** [comment]
@otraore I've seen tests fail (at this repo) because the demos weren't updated :P So I guess go test -v ./... does compile `main` packages.

**Comment by otraore** [comment]
@EtienneBruines You're right but turns out it's go get. Anyones free to add in the TrafficManager to the build in the meantime, if not I'll do it tomorrow or something.

**Comment by EtienneBruines** [comment]
Done in https://github.com/EngoEngine/engo/commit/e5a88d21b82a2fcab7048cf941b7bad02850c498.

**Pull request closed without merging**